### PR TITLE
Updated README.md to mention Redot engine, includes logo

### PR DESCRIPTION
**Pull request by EthanBoiLol**

### CODE REVIEW

File: README.md
Comment (JohnnyThunder2) [inline README.md]: The Redot Foundation doesn't exit yet... so we should probably cut this line for now.

**Review by JohnnyThunder2 — APPROVED** [review]
Looks good, I'll let the other maintainers comment this change once they've checked it too.

File: README.md
Comment (Salvakiya) [inline README.md]: These are empty. we should remove them unless they point to something valid.

**Review by Salvakiya — CHANGES_REQUESTED** [review]
lets change this real quick

**Comment by CCIGAMES** [comment]
you might want to make the logo for the readme longer so it looks nice
logo_outlined (1)
something like this with a cleaner R symbol/icon, i had to rip it out of the twitter png since theres no available svg/vector of the icon/logo

**Comment by Salvakiya** [comment]
it would be a good idea to get an SVG version of that logo. The git history should also be more descriptive.

**Comment by shasaur** [comment]
Guys, if you want to look like a serious fork, you should get this merged ASAP. It's literally just some text and a logo, it should not require comments and discussion. There are tons of people visiting this repo to see if you are serious about this, or if you are just playing around.

**Comment by Salvakiya** [comment]
@shasaur If we merge in changes they will be merged in correctly. Just changing the logo does not a Redot make. We dont have binaries yet anyway.

@kuligs2 Redot is not ready to be used yet anyway. As said before if we are going to maintain a fork we are going to do it correctly.

There is no rush to rebrand the repository. It is better to start off on the right foot to establish something sustainable. There will be many things we want to pull down from Godot and there will be things we contribute back to Godot.

**Comment by CCIGAMES** [comment]
I wouldn't contribute anything to godot

**Comment by finepointcgi** [comment]
path1

**Comment by Captain-Wet-Beard** [comment]
> **[Salvakiya](/Salvakiya) ** commented

I might want to do my own fork then tbh, there isn't much point doing a fork if the fork is just going to feed my work upstream, at that point I might as well just be contributing to the upstream directly.

File: README.md
Comment (EthanBoiLol) [reply README.md]: :+1:

**Comment by EthanBoiLol** [comment]
> you might want to make the logo for the readme longer so it looks nice logo_outlined (1) something like this with a cleaner R symbol/icon, i had to rip it out of the twitter png since theres no available svg/vector of the icon/logo

:+1:

**Comment by bayu-sw** [comment]
How about black and white logo

**Comment by RNavega** [comment]
Hi, suggesting a different design, drawn with spiros.

### PNG
redot_logo_spiro

### SVG (for Inkscape)
redot_logo_spiro

**Comment by Captain-Wet-Beard** [comment]
> Hi, suggesting a different design, drawn with spiros.
> 
> ### PNG
> redot_logo_spiro
> 
> ### SVG (for Inkscape)
> redot_logo_spiro

Wow, that's really nicely designed. Good job

**Comment by Naxdy** [comment]
> Hi, suggesting a different design, drawn with spiros.

That design is the most beautiful rendition of the letter "R" I've ever seen in my entire life. Good job!

**Review by Murly117 — APPROVED** [review]

**Comment by RNavega** [comment]
Wow, thanks guys! 😄

**Comment by filipworksdev** [comment]
There is a new red robot logo available now. Maybe wait a bit until new files are created with the new branding.

**Comment by EthanBoiLol** [comment]
Closing this as I'm redoing the entire README file

**Comment by apofex** [comment]
> Closing this as I'm redoing the entire README file

Waiting for results

**Comment by kuligs2** [comment]
> @kuligs2 Redot is not ready to be used yet anyway. As said before if we are going to maintain a fork we are going to do it correctly.

are you bringing the OG devs on this fork? How is it gonna be maintained when the OG devs still clinging on to the godot and commit on the original repo? Merging with the original and then branding as "better fork"?

**Comment by H2xDev** [comment]
Group 3 (1)

**Pull request closed without merging**
